core/src/main/java/org/apache/accumulo/core/util/compaction/CompactionServicesConfig.java

```diff
@@ -133,13 +133,11 @@ public CompactionServicesConfig(AccumuloConfiguration aconf) {
           if (eprop == null || aconf.isPropertySet(eprop)) {
             rateLimits.put(tokens[0], ConfigurationTypeHelper.getFixedMemoryAsBytes(val));
           }
-        } else if (!(tokens.length == 2 && tokens[1].equals("planner"))) {
+        } else if (tokens.length == 2 && tokens[1].equals("planner")) {
```

Comment:
Just to make sure I am understanding this correctly, is this ignoring the planner class when looking for planner options?

Reply:
Yes, because we've already processed the planner option on line 96. 

https://github.com/apache/accumulo/pull/3987/files#diff-ba8ebe6daae5454d3fd92a0f328ed7a6b40a5796994831c314f5ad0a6dd5aa8aR96

Since we aren't going to do anything with the planner option in this loop we just call `return` for this property and move on to the next property.